Fabric.Identity.API/Services/Azure/AzureActiveDirectoryClientCredentialsService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Fabric.Identity.API;
+using Fabric.Identity.API.Exceptions;
+using IdentityModel.Client;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Fabric.Identity.API.Configuration;
+using Microsoft.Extensions.Localization;
+

```

Comment:
It looks like most of this code for IdPSS functionality is directly pulled from there.

Reply:
Kind of. IdPSS had 4 different search endpoints, and those are reflected in the different IdP services here. We can probably remove some of the old functionality from Identity after this release.

Reply:
Yah I can see the changes from a nancy controller to an mvc like controller.